src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -679,27 +682,27 @@ def filter_out_esm_packages(self, packages, package_versions):
         """
         Filter out packages from the list where the version matches the UA_ESM_REQUIRED string.
         """
-        non_esm_packages = []
-        non_esm_package_versions = []
-        esm_packages = []
-        esm_package_versions = []
-        esm_packages_found = False
+        selected_packages = []
+        selected_packages_versions = []
+        ua_esm_required_packages = []
+        ua_esm_required_package_versions = []
+        ua_esm_required_packages_found = False
 
         for pkg, version in zip(packages, package_versions):
             if version != Constants.UA_ESM_REQUIRED:
-                non_esm_packages.append(pkg)
-                non_esm_package_versions.append(version)
+                selected_packages.append(pkg)
+                selected_packages_versions.append(version)
```

Comment:
What does selected_packages mean? non_esm_packages, etc seems more quickly intuitive here.

ua_esm_required_packages_* is fine

Reply:
Sure. Makes sense. Will update accordingly.

Reply:
Done